src/main/java/crawlercommons/robots/BaseRobotRules.java

```diff
@@ -60,12 +61,14 @@ public void setDeferVisits(boolean deferVisits) {
         _deferVisits = deferVisits;
     }
 
+    /** Add sitemap URL to rules if not a duplicate */
     public void addSitemap(String sitemap) {
         _sitemaps.add(sitemap);
     }
 
+    /** Get URLs of sitemap links found in robots.txt */
     public List<String> getSitemaps() {
```

Comment:
Maybe this method should return a Set instead of a List ?

Reply:
This would change the API which isn't nice, esp. after we released 1.0 - agreed that's not the ideal solution but I haven't found a better one.